app/src/main/java/com/nextcloud/talk/controllers/ConversationsListController.java

```diff
@@ -527,32 +558,19 @@ private void fetchData(boolean fromBottomSheet) {
                     }
 
                     adapter.updateDataSet(callItems, false);
+                    adapter.hideAllHeaders();  // TODO: the user still sees that hiding is done by animation. It's
+                    // still not hidden by default...
```

Comment:
Adding sections for the search works so far, but atm i have this problem:
On inital startup the header of the conversation list is shown for a second before it gets hidden (see video).

So https://github.com/davideas/FlexibleAdapter won't hide it by default, see this issue: https://github.com/davideas/FlexibleAdapter/issues/286
I could try the workaround https://github.com/davideas/FlexibleAdapter/issues/286#issuecomment-279675071 but i really would like to avoid that.

Also setting 
`adapter.setHeadersShown(false)`
before executing
`adapter.updateDataSet(callItems, false);`
has no effect.

Any ideas are welcome...

https://user-images.githubusercontent.com/2932790/150950863-8bcdfcbb-06c7-4be4-a0cf-e311d53bc2ab.mp4

Reply:
fyi @nickvergessen i'm stuck here atm. sections work so far, but for the bug in the video i have no idea...

Reply:
solved by handling 2 different lists for "normal" conversations. one with items that include header attribute and one without... not nice but does the job..